partner_address_street3/views/res_partner.xml

```diff
@@ -26,7 +26,7 @@
                 />
             </xpath>
             <xpath
-                expr="/form//field[@name='child_ids']/kanban/field[@name='street2']"
```

Comment:
if street2 has been removed from that view, maybe adding street3 is not needed at all? can you find the odoo commit of that change to understand the rationale? might have been to save space